### PR TITLE
cought with litmus props test propfind_invalid2

### PR DESCRIPTION
**Pull request by DeepDiver1975**

### CODE REVIEW

**Comment by staabm** [comment]
lets wait for a release of sabre/xml which contains sabre-io/xml#149 so we can also get the fix for the problem into this PR.

File: composer.json [hunk 1]
```diff
@@ -17,7 +17,7 @@
         "php": ">=7.0.0",
         "sabre/vobject": "^4.2.0-alpha1",
         "sabre/event" : "^5.0",
-        "sabre/xml"  : "^2.0.0",
+        "sabre/xml"  : "^2.1.1",
```
Comment (staabm) [inline composer.json]: we shouldn't rais this minimum version. composer will automatically pick the latest version which is compatible.

the version mentioned in the composer.json of our libs should be as low as possible, to support the widest range possible.

Comment (DeepDiver1975) [reply composer.json]: but if we know that 2.1.0 is broken - we better force this dependency. Otherwise users will run into the same issue and finding out about the reason is not trivial.

I prefer to be better safe then sorry ...

Comment (staabm) [reply composer.json]: every release of a software product will fix "some things" but only few of them will affect most users.
its not our job to tell people to use the latest versions of the used libraries.
in case we are too strict, people might even get a "composer is not able to find a compatible set of depdencies" error, so they can't use the software at all.

I feel this as a good general rule.

in our concrete case here the fix in 2.1.1 is feels like a edge-case and most users will not produce xml like that.

do you think error in the namespace are a common problem?

**Comment by DeepDiver1975** [comment]
@staabm I reverted the change to composer.

File: tests/Sabre/DAV/Xml/ServiceTest.php [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+namespace Sabre\DAV\Xml;
+
+use PHPUnit\Framework\TestCase;
+
+class ServiceTest extends TestCase
+{
+
+    /**
+     * @expectedException \Sabre\Xml\LibXMLException
```
Comment (staabm) [inline tests/Sabre/DAV/Xml/ServiceTest.php]: we need to skip this test when `LOWEST_DEPS` ENV var is empty

Comment (DeepDiver1975) [reply tests/Sabre/DAV/Xml/ServiceTest.php]: Hint on how to do this is welcome ..... but I have to say that this is fishy from my pov ....

Please note that the issue we are trying to fix is a regression - it was working properly in previous dav versions ......

Comment (staabm) [reply tests/Sabre/DAV/Xml/ServiceTest.php]: hmm ok, this is new information. shouldn't we backport the fix into 2.0.x then?

Comment (DeepDiver1975) [reply tests/Sabre/DAV/Xml/ServiceTest.php]: let me see what I can do ......

Comment (staabm) [reply tests/Sabre/DAV/Xml/ServiceTest.php]: cool. thank you very much.

merged and restarted the travis build.

**Comment by staabm** [comment]
thx

**Comment by DeepDiver1975** [comment]
We did not yet sort out travis failure for lowest_deps ....

**Comment by staabm** [comment]
Will be fixed in 
https://github.com/sabre-io/dav/pull/1095
